india_compliance/gst_india/constants/custom_fields.py

```diff
@@ -48,6 +48,214 @@
 ]
 
 CUSTOM_FIELDS = {
+    # Subcontracting: Tax Fields
+    "Subcontracting Order": [
+        {
+            "fieldname": "section_break_taxes",
+            "label": "Taxes",
+            "fieldtype": "Section Break",
+            "insert_after": "total",
+            "depends_on": "purchase_order",
+        },
+    ],
+    "Subcontracting Receipt": [
+        {
+            "fieldname": "section_break_taxes",
```

Comment:
Job worker challan number and date.
Document number for waybill should be this in case of receipt (similar to purchase)